functionnal_tests/cypress/integration/app_file/video_play_spec.js

```diff
@@ -0,0 +1,45 @@
+import { PAGES as p } from '../../support/urls_commands'
+
+const fileTitle = 'FileTitle'
+const fullFilename = 'video.mp4'
+const contentType = 'video/mp4'
+
+let workspaceId, contentId
+
+describe('Paly a video file', () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+      cy.createFile(fullFilename, contentType, fileTitle, workspaceId)
+        .then(newContent => {
+          contentId = newContent.content_id
+        })
+    })
+  })
+
+  beforeEach(function () {
+    cy.loginAs('administrators')
+    cy.visitPage({ pageName: p.CONTENT_OPEN, params: { contentId } })
+  })
+
+  afterEach(function () {
+    cy.cancelXHR()
+  })
+
+  it('should display the full video in fullscreen', () => {
+    cy.contains('[data-cy=FilenameWithBadges__label]', fileTitle)
+    cy.get('[data-cy=dropdownContentButton]').click()
+    cy.get('[data-cy=popinListItem__playVideo').click()
+    cy.get('#videoWrapperDiv > video').should('be.visible')
```

Comment:
does this validates that the entire video player is visible?

Reply:
no, I didn't find a way to check this with cypress, so I check at least if the video and the message are visible

Reply:
okok

Reply:
could you check that the video width and height are inferior to the viewport width and height?
that would validate the player controls are visible but it would validate the player is smaller than the viewport, therefore the player should be entirely visible so its controls too

Reply:
Yes, I'm going to try :D good idea